spec/expectations_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'spec_helper'
+
+describe RubyExpectationsHook do
+  def req(expectations, content)
+    struct expectations: expectations, content: content
+  end
+
+  def compile_and_run(request)
+    runner.run!(runner.compile(request))
+  end
+
+  let(:runner) { RubyExpectationsHook.new(mulang_path: './bin/mulang') }
+  let(:result) { compile_and_run(req(expectations, code)) }
+
+  context 'smells' do
+    let(:code) { 'x = 4' }
+    let(:expectations) { [] }
+
+    it { expect(result).to eq [{binding: 'x', inspection: 'HasTooShortBindings'}] }
+  end
+
+
+  context 'expectations' do
+    let(:code) { 'pepita = Object.new' }
```

Comment:
That is true, but It was not my intention to check that scenario here - I am actually looking for variable declararions instead of object declarations. 

Please check these tests: https://github.com/mumuki/mulang-ruby/blob/master/spec/mulang/ruby_spec.rb#L117